### ISSUE TITLE
ERROR  Pinceau could not resolve this configuration file [v1.9.0]

### ISSUE THREAD

**Issue by PhE** [nuxt-themes/docus#831]
With the 1.9.0 release, I've got this error with a fresh new install :

``` shell
# npx nuxi@latest init docs88 -t themes/docus
# cd docs88
# yarn install
# yarn dev
``` 
The `pinceau` shows up :
![image](https://user-images.githubusercontent.com/1107924/221672785-5a188d45-eefe-4ec9-9ea1-d2014ac7a4b6.png)

**Comment by Barbapapazes** [comment]
Yes, same here, I create an issue in pinceau https://github.com/Tahul/pinceau/issues/66

**Comment by PhE** [comment]
Fixed with the docus 1.9.1 release

Thanks.
